Ephemeral model updates (#8796)
fishtown-analytics/docs.getdbt.com: website/docs/best-practices/materializations/materializations-guide-5-best-practices.md

      +materialized: view
```

### Ephemeral intermediate models

Intermediate models sit between staging and marts, breaking up complex transformations into manageable pieces. Consider these aspects when choosing their materialization:

- 🚫 Intermediate models are not accessed directly by end users. They exist to simplify mart logic.
- 🧩 They serve as building blocks that get referenced by marts or other intermediate models.
- 👻 This makes them ideal candidates for ephemeral materialization, which doesn't create objects in your warehouse.

> Ephemeral models are interpolated as CTEs into the models that reference them. This keeps your warehouse clean and avoids cluttering it with models that aren't meant for direct querying:

Reviewer: ```suggestion
Ephemeral models are interpolated as <Term id="cte">CTE</Term> into the models that reference them. This keeps your warehouse clean and avoids cluttering it with models that aren't meant for direct querying:
```

```yaml
models:
  jaffle_shop:
    staging:
      +materialized: view
    intermediate:
      +materialized: ephemeral
```

> :::tip When to avoid ephemeral

Reviewer: ```suggestion
:::tip When to avoid ephemeral models
```

> :::tip When to avoid ephemeral

Reviewer: ```suggestion
:::tip When to avoid ephemeral models
```
